src/backend/joanie/core/views.py

```diff
@@ -45,77 +62,55 @@ class DebugMailSuccessPaymentViewTxt(DebugMailSuccessPayment):
     template_name = "mail/text/order_validated.txt"
 
 
-class DebugCertificateDefinitionTemplateView(TemplateView):
+class DebugPdfTemplateView(TemplateView):
```

Comment:
If all the views defined in this PR are made for ony for debugging purpose, we should create a debug app and move them in this app. The urls will be included only if `settings.DEBUG` is `True`.

Reply:
Okay, i think it's a good idea. Do we implement your suggestion into a new PR ? Or should I carry on with this one ?

Reply:
You can create an issue for that

Reply:
The [issue](https://github.com/openfun/joanie/issues/560) has been created